docs/open-source-project/release-notes/tizen-4-0-m2.md

```diff
@@ -451,11 +451,11 @@ The Tizen 4.0 Public M2 release provides developers with the Tizen kernel, devic
     - iptables has been upgraded from 1.4.21 to 1.6.1.
     - openvpn has been upgraded from 2.3.2 to 2.4.2.
 - Bluetooth
-  - New features and APIs:
-    - The C# Public OPP Client API has been added.
-    - The C# Public OPP Server API has been added.
+  - New features and API set:
+    - The C# OPP Client API has been added.
```

Comment:
From line 455 to line 4556 : Why "Public" is removed?

Reply:
Added